### PR TITLE
added animation to the gradients on home page

### LINKED ISSUE #146: On the homePage hero section animate the gradients :)
### Would you like to have a new feature? Please describe.

I would like to add (have) an animation to the gradients on the home pages, there are various text on that page that are highlightes with a beautiful looking gradient, I would be obliged if I can take that gradient to next level using an infinite animate loop !

- how it is looking : (beautiful but simple and static)
image

- how it will look  : (beautiful and alive)

https://user-images.githubusercontent.com/68517592/203650618-36a1c848-a0d1-4bee-9309-1d4e88e18dcd.mp4

### How would this feature be beneficial for the project

- I must firsly mention that it will surely increase the overall look of the page and it will give the highlighted gardient text a different level of cool pallete
- Other than the enhancement of look honestly I do not see any other benefits for this design implementation

### Additional context

I had already updated the necessary files of this repo on my forked branch and would like to submit that for a PR if I get the permission !! Thank you

### PR DESCRIPTION
**Pull request by arian0zen**
## Fixes Issue

This PR fixes the following issues:

closes #146 

## Changes proposed

A simple looking infinite gradient animation on the gradient texts

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots 

https://user-images.githubusercontent.com/68517592/203653417-aa7f2e25-2040-4311-af5a-33f3b56750af.mp4

### CODE REVIEW

File: src/styles/Founder.css [hunk 1]
```diff
@@ -10,12 +26,15 @@
 }
 .founder_span
 {
-    background: #ee0979;  /* fallback for old browsers */
+    animation: background-pan 3s linear infinite;
+    background: linear-gradient(to right, var(--lightSky), var(--lightGreen), var(--lighterGreen), var(--lightSky));
+    background-size: 200%;
     height: 1.5em;
-    background-image: linear-gradient(90deg, rgba(64,148,236,1) 0%, rgba(6,214,160,1) 100%);
+    /* background-image: ; */
```
Comment (siddhigate) [inline src/styles/Founder.css:33]: You can delete the comment.

File: src/styles/Header.css [hunk 2]
```diff
@@ -29,12 +44,16 @@
...
 }
 .header_free{
-    background: #ee0979;  /* fallback for old browsers */
+   
+    animation: background-pan 3s linear infinite;
+    background: linear-gradient(to right, var(--lightSky), var(--lightGreen), var(--lighterGreen), var(--lightSky));
+    background-size: 200%;
     height: 1.5em;
-    background-image: linear-gradient(90deg, rgba(64,148,236,1) 0%, rgba(6,214,160,1) 100%);
+    /* background-image: ; */
```
Comment (siddhigate) [inline src/styles/Header.css:52]: You can delete this comment.

**Review by siddhigate — COMMENTED** [review]
LGTM!

**Review by arian0zen — COMMENTED** [review]
deleted those unnecessary comments !  @siddhigate  check that :)

**Review by siddhigate — APPROVED** [review]
Good to go 🚀

**Comment by AnuragThePathak** [comment]
@arian0zen as a word of advice, prefer resolving the conversation once you've made the changes.

**Comment by arian0zen** [comment]
> @arian0zen as a word of advice, prefer resolving the conversation once you've made the changes.

heyyy, did that !!
I did not know I have to resolve those after suggested changes, thanksssss 🔢

**Pull request merged**